cmd/network-passt-binding/domain/configurator_test.go

```diff
@@ -0,0 +1,389 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package domain_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	vmschema "kubevirt.io/api/core/v1"
+
+	domainschema "kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+
+	"kubevirt.io/kubevirt/cmd/network-passt-binding/domain"
+)
+
+var _ = Describe("pod network configurator", func() {
+	Context("generate domain spec interface", func() {
+		DescribeTable("should fail to create configurator given",
+			func(ifaces []vmschema.Interface, networks []vmschema.Network) {
+				_, err := domain.NewPasstNetworkConfigurator(ifaces, networks, domain.NetworkConfiguratorOptions{})
+
+				Expect(err).To(HaveOccurred())
+			},
+			Entry("no pod network",
+				nil,
+				[]vmschema.Network{{Name: "default", NetworkSource: vmschema.NetworkSource{Multus: &vmschema.MultusNetwork{}}}},
+			),
+			Entry("no corresponding iface",
+				[]vmschema.Interface{{Name: "not-default", Binding: &vmschema.PluginBinding{Name: "passt"}}},
+				[]vmschema.Network{*vmschema.DefaultPodNetwork()},
+			),
+			Entry("interface with no passt binding method",
+				[]vmschema.Interface{{Name: "default", InterfaceBindingMethod: vmschema.InterfaceBindingMethod{Bridge: &vmschema.InterfaceBridge{}}}},
+				[]vmschema.Network{*vmschema.DefaultPodNetwork()},
+			),
+			Entry("interface with no passt binding plugin",
+				[]vmschema.Interface{{Name: "not-default", Binding: &vmschema.PluginBinding{Name: "passt"}}},
```

Comment:
This *is* a passt binding, but the name is wrong... same as the second scenario above.